CONTRIBUTING.md

```diff
@@ -198,6 +198,26 @@ HEADLESS=false SLOW_MO=500 npm run wtest
 BROWSER=chromium node --inspect-brk test/test.js
 ```
 
+- When should a test be marked with `skip` or `fail`?
```

Comment:
I did not comment about how these interact with CI/CD of an MR passing/failing. It may be helpful to add that, too.